test_data/rules/no_leading_underscores_for_local_identifiers.dart

```diff
@@ -5,6 +5,15 @@
 // test w/ `dart test -N no_leading_underscores_for_local_identifiers`
 
 
+///https://github.com/dart-lang/linter/issues/3126
+void fn0() {
+  for (var _ in []) { } // OK
+  var _ = g(); //OK
+  for (var _ = 0; _ < 1; ++_) { } // OK
```

Comment:
It would be good to have at least one test with a name that is multiple underscores.

Reply:
Done. Thanks.